chapter_explainability/overview.md

```diff
@@ -0,0 +1,64 @@
+
+**Stakeholders of explanation methods:**
+The stakeholders can be end-users who are directly affected by the AI decisions, AI practitioners who design and implement the AI models, domain experts who will refer to the explanation methods when making decisions, business owners who are responsible for ensuring AI systems to be aligned with corporate strategies, and governments who should regulate the usage of AI algorithms.
+
+**Inherently interpretable models vs. post-hoc explanation methods:**
+While some conventional machine learning models :cite:`Bishop.2006` such as linear regression, logistic regression, naive Bayes classifier, k-nearest neighbors, decision trees, decision set, and generalized additive models (GAMs) :cite:`Hastie.Tibshirani.2017` are inherently explainable, most black-box models (e.g., deep neural networks, random forests :cite:`Breiman.2001`, support vector machines :cite:`Scholkopf.Smola.2002`, XGBoost :cite:`Chen.Guestrin.2016`, etc.) which are often demonstrated to be more effective in real-world applications are challenging to interpret. There is usually a tradeoff between the predictive accuracy and how easy the model is to interpret.
+
+**Global explanations vs. local explanations:**
+Pertaining the scope of explanations, we can classify explanations into global explanations and local explanations. Global explanation methods describe the average behavior of a black-box model. A typical global explanation method is the global feature importance plot which displays how much impact each feature has on average for model predictions. On the contrary, local explanation methods are centered around the prediction of each instance and focus on explaining how a specific prediction is obtained for an individual instance. For example, we will know which pixels make an image be classified as a bird via local explanation methods for image classifiers. Local explanations are helpful to vet if individual predictions are being made for the right reasons.
+
+**Model-agnostic methods vs. model-specific methods:**
+Based on the applicability of explanation methods, we have model agnostic explanation methods and model-specific explanation methods. The former is more flexible and can be applied to any black-box model regardless of its structure, while the latter is limited to specific model classes.
+
+
+## Desired Properties of Explanations
+There are no standard evaluation measures on explainability. Still, the following dimensions, which might be challenging to quantify rigorously, can be considered as guiding intuitions when developing or evaluating explanation methods:
+
+* Fidelity (or Faithfulness): How well does the explanation model capture the black box models. For instance, if we use a surrogate model to mimic a black-box model, we can check if it can give the same outputs as the original model. We can also measure the fidelity by checking whether the prediction changes if the supposedly important features are flipped.
```

Comment:
"surrogate model" is an undefined jargon. Consider adding a brief explanation such as "surrogate models (explanations/definitions here)"